apps/src/pythonlab/pyodideWebWorker.js

```diff
@@ -12,11 +12,17 @@ async function loadPyodideAndPackages() {
   });
 }
 
-const pyodideReadyPromise = loadPyodideAndPackages();
+let pyodideReadyPromise = null;
+export async function initializePyodide() {
+  if (pyodideReadyPromise === null) {
+    pyodideReadyPromise = loadPyodideAndPackages();
+  }
+  await pyodideReadyPromise;
```

Comment:
Remind my brain that's async/await rusty: If we've already awaited this promise once (on initial load) and it's resolved, this is functionally just a return statement, right?

Reply:
yep if it's already resolved it will return immediately